.github/workflows/jiraCreateIssue

```diff
@@ -0,0 +1,28 @@
+on: 
+  issues:
+    types: [opened]
+
+name: Jira gemeente team issue creation from git issue
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Jira Example
+    steps:
+    - name: Login
+      uses: atlassian/gajira-login@v3
+      env:
+        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+    - name: create
+      id: create
+      uses: atlassian/gajira-create@v3
```

Comment:
Can you change the version tag to a specific commit tag instead? This is to harden against supply chain attacks. Commit tags are immutable, version tags can overwritten.

Reply:
done